packages/hooks/src/useIsSsr.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+
+export default function useIsSsr() {
```

Comment:
Could just make this `useRenderingEnv` or something like similar, but this seemed easier.

Reply:
is `use` referring to the `user`?

Reply:
I think that's just that hook syntax

Reply:
I personally don't like the `useIs...` naming for hooks. I like `useRenderEnv` or something

Reply:
I hate it as well lol. I think the only downside of some `useEnv` alternative is that I would expect that to return an env enum-like string as opposed to just a boolean. How about `useSsrCheck`? The might read better `const isSsr = useSsrCheck()`

Reply:
I like `useSSRCheck` 👍